.github/pull_request_template.md

```diff
@@ -1,39 +1,33 @@
 ## Description
+<!--- A detailed explanation of the changes in your PR. Feel free to remove this section if the title of your PR is sufficiently descriptive. -->
 
-A detailed explanation of the changes in your PR.
-
-Feel free to remove this section if it is overkill for your PR, and the title of your PR is sufficiently descriptive.
 
 ## Checklist
-- [ ] Investigated and inspected CI test results
-- [ ] Unit test and regression tests added
-- [ ] Evaluated and added CHANGELOG entry if required
-- [ ] Determined and documented upgrade steps
-- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))
-
-If any of these don't apply, please comment below.
+All items *must be* checked:
+- [ ] CI results are inspected
+- [ ] CHANGELOG is updated **OR** not needed
+- [ ] Documentation PR is created and linked above **OR** not needed
```

Comment:
I think, @sthadka and @tommartensen might have considered making it required for all checkboxes to be checked in order to merge the PR. Given that GitHub does not support radio buttons, we can have more items and rely on engineers deleting lines. Deleting is easier than striking through.

I mean this:
```suggestion
All three items *must be* checked:
<!-- Remove conflicting items that won't be checked -->

- [ ] CI results are inspected
- [ ] CHANGELOG is updated
- [ ] I believe CHANGELOG update is not needed
- [ ] Documentation PR is created and linked above
- [ ] Documentation is not needed
```

A valid state would be:

> All three items *must be* checked:
> <!-- Remove items that don't apply -->
> 
> - [x] CI results are inspected
> - [x] I believe CHANGELOG update is not needed
> - [x] Documentation PR is created and linked above

A clearly invalid state includes contradicting items checked (I am a teapot):

> All three items *must be* checked:
> <!-- Remove items that don't apply -->
> 
> - [x] CI results are inspected
> - [x] CHANGELOG is updated
> - [x] I believe CHANGELOG update is not needed
> - [x] Documentation PR is created and linked above
> - [x] Documentation is not needed

Reply:
> I think, @sthadka and @tommartensen might have considered making it required for all checkboxes to be checked in order to merge the PR.

Yes, I have a spike workflow that does exactly that. It uses `Shopify/task-list-checker` action and uses `N/A` and `POST-MERGE` to detect items that can be skipped.